htdocs/expedition/card.php

```diff
@@ -268,6 +270,7 @@
 		$object->ref_customer = GETPOST('ref_customer', 'alpha');
 		$object->model_pdf = GETPOST('model');
 		$object->date_delivery = $date_delivery; // Date delivery planned
+		$object->date_shipping = $date_shipping; // Date delivery planned
```

Comment:
Hi @altairis-noe 
Please change the comment to // Sending date

Reply:
Thanks for the review @sonikf. It’s done!

Reply:
https://github.com/Dolibarr/dolibarr/blob/71cde0e95f8131b92edc7e25c49b5d7ecc890902/htdocs/compta/facture/class/facture.class.php#L127
Can you please change this comment too as it is misleading and makes no sense, planned and expected date has nothing to do with actual date of start